ddspipe_yaml/include/ddspipe_yaml/yaml_configuration_tags.hpp

```diff
@@ -122,6 +122,12 @@ constexpr const char* TIMESTAMP_MILLISECONDS_TAG("milliseconds"); //! TODO: add
 constexpr const char* TIMESTAMP_MICROSECONDS_TAG("microseconds"); //! TODO: add comment
 constexpr const char* TIMESTAMP_NANOSECONDS_TAG("nanoseconds"); //! TODO: add comment
 
+// Custom routes related tags
+constexpr const char* ROUTES_TAG("routes"); //! Forwarding routes
+constexpr const char* ROUTES_SRC_TAG("src"); //! Route source
```

Comment:
I would write "Route's source participant" and "Route's destination participants"